src/events/crawler/scrape-data/normalize-locations.js

```diff
@@ -11,6 +22,38 @@ const normalizeLocations = args => {
       if (location.country === 'USA') {
         // Normalize states
         location.state = geography.toUSStateAbbreviation(location.state);
+
+        if (location.county) {
+          // Find in fips
+          if (location.feature && location.feature._aggregatedLocations) {
+            let aggregatedCounty = [];
+            let fipsFound = true;
+            for (let aggregatedLocation of location.feature._aggregatedLocations) {
+              let countryLevelId = findCountryLevelID(aggregatedLocation);
+              if (countryLevelId) {
+                aggregatedCounty.push(countryLevelId);
+              }
+              else {
+                fipsFound = false;
+                log.error('❌ Failed to find FIPS code for subset of combined region %s, %s', aggregatedLocation.county, aggregatedLocation.state);
+              }
+            }
+            if (fipsFound) {
+              // I have no idea if this is useful at all, but it sort of matches what we discussed
+              location.countyId = aggregatedCounty.join('+');
```

Comment:
Yes, super useful!

Reply:
Oh, no we have to set it to the county "name". It'll process it to id at the last step.

Reply:
I think you're right, especially with the other comment on combining names.